README.md

```diff
@@ -5,7 +5,7 @@
   ![Platform](https://img.shields.io/badge/Platform-Linux%20%7C%20macOS-informational)
   [![Python](https://img.shields.io/badge/Python-3.7%20%7C%203.8%20%7C%203.9%20%7C%203.10-informational)](https://www.python.org/)
   [![Qiskit](https://img.shields.io/badge/Qiskit-%E2%89%A5%200.39.0-6133BD)](https://github.com/Qiskit/qiskit)
-  [![Qiskit Nature](https://img.shields.io/badge/Qiskit%20Nature-%E2%89%A5%200.4.4-6133BD)](https://github.com/Qiskit/qiskit-nature)
```

Comment:
Was the color change intentional?  The colors of Qiskit and Qiskit Nature at https://github.com/Qiskit-Extensions/circuit-knitting-toolbox/blob/remove-deprecated-nature-code/README.md appear just a little bit different for me.

Reply:
Indeed they are, at a glance on the shields page, that looked like the color that matched "blueviolet". I'll take another look